docs/btcli.md

```diff
@@ -6171,18 +1712,21 @@ btcli sudo get --netuid 1
 **Usage**:
 
 ```console
-btcli subnet hyperparameters [OPTIONS]
+btcli subnets hyperparameters [OPTIONS]
 ```
 
 **Options**:
 
-- `--network, --subtensor.network, --chain, --subtensor.chain_endpoint TEXT`: The subtensor network to connect to. Default: finney.
-- `--netuid INTEGER`: The netuid of the subnet in the network, (e.g. 1).
-- `--quiet`: Display only critical information on the console.
-- `--verbose`: Enable verbose output.
-- `--help`: Show this message and exit.
+| Option                                                                      | Type    | Description                                           |
+| --------------------------------------------------------------------------- | ------- | ----------------------------------------------------- |
+| `--network`, `--subtensor.network`, `--chain`, `--subtensor.chain_endpoint` | TEXT    | The subtensor network to connect to. Default: finney. |
+| `--netuid`                                                                  | INTEGER | The netuid of the subnet in the network, (e.g. 1).    |
+| `--quiet`                                                                   |         | Display only critical information on the console.     |
+| `--verbose`                                                                 |         | Enable verbose output.                                |
+| `--json-output`, `--json-out`                                               |         | Outputs the result of the command as JSON.            |
+| `--help`                                                                    |         | Show this message and exit.                           |
 
-### `btcli subnet list`
+### `btcli subnets list`
 
```

Comment:
there's a rendering issue with that list under this header (not something you caused)
<img width="1013" alt="Screenshot 2025-07-09 at 8 02 49 AM" src="https://github.com/user-attachments/assets/ed99e8d1-e6f4-4c52-ab49-f453568801f1" />